work_2.html

```diff
@@ -32,7 +32,81 @@ <h3>Filter by age</h3>
 // 2. При выборе соответствующего option осуществить вывод
 //    соответствующих пользователей в элемент <div id="res"></div>
 
+'use strict';
 
+const usersSelect = document.getElementById('users-select');
+const res = document.getElementById('res');
+
+let users = [
+ {id: 1, name: "John", age: "20"},
+ {id: 2, name: "Sasha", age: "30"},
+ {id: 3, name: "Bill", age: "40"},
+ {id: 4, name: "Ashley", age: "20"},
+ {id: 5, name: "Rachel", age: "40"},
+ {id: 6, name: "Tom", age: "30"},
+ {id: 7, name: "Steve", age: "30"},
+ {id: 8, name: "Jim", age: "40"},
+ {id: 9, name: "Willy", age: "20"}
+];
+
+addToSelectOptins();
+start();
+
+// добавление значений в селект
+function addToSelectOptins () {
+	let arr = [];
+	users.forEach(function(item){
+		let pushed = arr.push(item.age);
+	});		
+	let arrSort = unique(arr);	
+	arrSort.forEach(function(item){
+		let newOption = new Option(item);
```

Comment:
don`t see realization of Option(

Reply:
Sorry i forgot about that https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionElement/Option